### DISCUSSION TITLE
Stylus buttons not activating

### DISCUSSION THREAD

**Discussion by traemand** [xournalpp/xournalpp#4387] [Q&A]
Hi,

I've recently switched back to windows 10 from Ubuntu on my surface pro 5th gen. I've had no issues using Xournal++ while on Ubuntu, but on Windows i'm having issues. I set my button 1 to Hand tool and button 2 to Eraser, but none of them are being activated when i push them. The same setting was working on Ubuntu. I'm using an older Intuous Art Medium.

Any help is appreciated!

**Comment by miltonism** [comment]
Did you set the buttons from the settings in Xournal++ or from the Intuous driver/software? If from Xournal++, it doesn't work. I use a Huion Kamvas (drawing tablet), and it doesn't work when set from the settings. I use the Huion software to set what keys/key-combo the pen buttons do, and so when I click one, it's set to the brush shortcut (Shft+Ctrl+P), and the other is set to the eraser shortcut (Shft+Ctrl+E).

**Comment by rolandlo** [comment]
@traemand What version of Xournal++ do you use? Make sure you got a recent version. Also try toggling Windows ink in the Windows settings.
